target/scripts/startup/setup.d/dmarc_dkim_spf.sh

```diff
@@ -96,9 +96,17 @@ function _setup_policyd_spf
 
 policyd-spf    unix  -       n       n       -       0       spawn
     user=policyd-spf argv=/usr/bin/policyd-spf
+EOF
+
+  sedfile -i -E \
+    's|^(smtpd_recipient_restrictions.*reject_unauth_destination)(.*)|\1, check_policy_service unix:private/policyd-spf\2|' \
+    /etc/postfix/main.cf
+  cat >>/etc/postfix/main.cf <<EOF
+
+# SPF policy settings
+policyd-spf_time_limit = 3600
```

Comment:
Comment doesn't really need to be embedded into the `main.cf` for single setting. This suggestion would need to remove the `EOF` directly after it though.

```suggestion
  # SPF policy settings
  postconf 'policyd-spf_time_limit = 3600'
```

or

```suggestion

  cat >>/etc/postfix/main.cf <<EOF
# SPF policy settings
policyd-spf_time_limit = 3600
```